components/automate-gateway/api/event_feed/response/event.proto

```diff
@@ -11,12 +11,18 @@ message Events {
 }
 
 message Event {
+  // Type of event (cookbook, role, etc).
   string event_type = 1;
+  // Type of task for the event (create, update, delete).
   string task = 2;
+  // Start time of the event.
   google.protobuf.Timestamp start_time = 3;
   string entity_name = 4;
+  // Requestor type on the event record.
   string requestor_type = 5;
+  // Requestor name on the event record.
   string requestor_name = 6;
+  // Hostname from which the record was gathered.
   string service_hostname = 7;
   string start_id = 8;
```

Comment:
start id? what is this?

Reply:
The "start_id" and "end_id" are only different when it is a grouped/collapsed event. For grouped events, we need to know the start date, end date, start ID, and end ID to request all the events in the grouped to show in the right side panel (shown below).

This is used similarly to the above when requesting events. 

![image](https://user-images.githubusercontent.com/1679247/85794914-3ec73080-b6ec-11ea-8194-9d6606713476.png)